app/src/main/java/com/trainstation/frettirtest2/MyViewPagerFragment2.java

```diff
@@ -83,16 +78,4 @@ public void run() {
 
         return cellDataList;
     }
-
-
-
-/*
-
-    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState){
```

Comment:
unused, and the onCreateView is already here: https://github.com/helgio/frettaApp/pull/2/files#diff-e85e3756c0867310389eb998516620ceR32 so I am removing this code